demo/debug_camera.hpp

```diff
@@ -83,6 +83,17 @@ class DebugCamera : public wr::CameraNode
 		}
 	}
 
+	const float m_scroll_speed = 0.25f;
+
+	void MouseWheel(int amount)
+	{
+		float percent = (float) GET_WHEEL_DELTA_WPARAM(amount) / WHEEL_DELTA * m_scroll_speed;
```

Comment:
Check if you hold rmb because other wise scrolling in imgui will scroll in the demo.